crypto/thread/build.info

```diff
@@ -14,7 +14,7 @@ ELSE
   IF[{- !$disabled{quic} -}]
     SOURCE[../../libssl]=$THREADS_ARCH
   ENDIF
-  $THREADS=api.c
+  $THREADS=api.c arch/thread_win.c
```

Comment:
shall we also include `arch/thread_posix.c`? just question I have not tried fiddling with no-thread/no-thread-pool options yet.

thanks.

Reply:
The rcu locks implementation does not depend on arch/thread_posix.c.